kickoff/jsonexport.py

```diff
@@ -277,3 +280,54 @@ def json_exports_txt():
     response = make_response(render_template("json_exports.txt", jsonFiles=jsonFiles))
     response.mimetype = "text/plain"
     return response
+
+
+def get_releases_for_json(product):
+    releases = getFilteredReleases(product, ["major", "stability", "esr", "dev"], detailledInfo=True)
+    listReleases = defaultdict()
+    for r in releases:
```

Comment:
if you already know what's inside the releases, then you can directly unpack the information here:
```
for version, date, version, etc in releases:
  # directly use the variables then.
```